admin/mirage/handlers/admin-members.js

```diff
@@ -0,0 +1,13 @@
+export function createAdminMember(schema, request) {
+  const payload = JSON.parse(request.requestBody);
+  const attributes = { ...payload.data.attributes };
```

Comment:
je trouve ca plus complexe que necessaire, suggestion 👇 
```suggestion
  const { email, role } = payload.data.attributes;
```